system_test.go

```diff
@@ -12,6 +13,8 @@ import (
 	"github.com/rogpeppe/go-internal/testscript"
 )
 
+var updateFlag = flag.Bool("update", false, "update testscript scripts to correspond with actual output")
+
```

Comment:
Where is this used?

Reply:
It can be used when running the tests if the test output has changed, to avoid the minor tedium of copying/pasting command output into the scripts (for example if output field order changes).